src/config.ts

```diff
@@ -49,16 +50,16 @@ export interface Config {
   search: boolean; // default to false
 }
 
-export async function readConfig(configPath?: string, root?: string): Promise<Config> {
+export async function readConfig(configPath?: FilePath, root?: FilePath): Promise<Config> {
   if (configPath === undefined) return readDefaultConfig(root);
-  const importPath = join(process.cwd(), root ?? ".", configPath);
+  const importPath = "file:///" + filePathToUrlPath(fileJoin(process.cwd(), root ?? ".", configPath));
```

Comment:
I'm not quite sure what you mean. The original version already called `join(process.cwd(), ...)`, so the `importPath` would be an absolute path. Windows _does_ insist on `file://` protocol here, but I think there is a relative form we could use.

Reply:
Ah yes it was already absolute. As a nit if we must start with`file:///` we should add only two `//` since the third one is the first char of `process.cwd()`. Currently I see 4 of them :)

`file:////Users/fil/Source/framework/observablehq.config.js`